DiscordBot/Commands/EventCommands.cs

```diff
@@ -251,10 +251,9 @@ private static async Task SendSignupMessage(CommandContext context, int eventKey
             var eventsSheetsService = context.Dependencies.GetDependency<IEventsSheetsService>();
             var discordEvent = await eventsSheetsService.GetEventAsync(eventKey);
 
-            var userResponseDictionary = await eventsSheetsService.GetSignupsByUserAsync(eventKey);
             var optionsList = await eventsSheetsService.GetSignupsByResponseAsync(eventKey);
 
-            var signupEmbed = GetSignupEmbed(discordEvent, userResponseDictionary, optionsList);
+            var signupEmbed = GetSignupEmbed(discordEvent, optionsList);
```

Comment:
I'm not sure `optionsList` is the most descriptive name. What about `signupsByResponse` which matches the service method name? 🙂